relay/src/lib.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright 2023 RISC Zero, Inc.
```

Comment:
I hit a few clippy warnings here, might be worth doing a clippy pass on things.

Reply:
I've fixed a few of them, and the remaining warnings are about using `expect` rather than throwing errors. Since there's no error handling in place, I think panicking is appropriate for this app, so I've disabled that check.